fxa/__main__.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import argparse
+import getpass
+import os
+import re
+import sys
+
+from fxa.errors import ClientError
+from fxa.tools.bearer import get_bearer_token
+from fxa.tools.browserid import get_browserid_assertion
+from fxa.tools.fxa import create_new_fxa_account
+
+FXA_API_URL = "https://api-accounts.stage.mozaws.net/v1"
+FXA_OAUTH_URL = "https://oauth.stage.mozaws.net/v1"
+DEFAULT_CLIENT_ID = "5882386c6d801776"  # Firefox dev Client ID
+
+
+def main(args=None):
+    """The main routine."""
+    if args is None:
+        args = sys.argv[1:]
+
+    parser = argparse.ArgumentParser(description="PyFxA commands")
+    parser.add_argument('--bearer',
+                        help='Generate a Bearer token',
+                        dest='bearer',
+                        action='store_true')
+
+    parser.add_argument('--browserid', '--bid',
+                        help='Generate a BrowserID assertion',
+                        dest='browserid',
+                        action='store_true')
+
+    parser.add_argument('--create', '-c',
+                        help='Create a new user',
+                        dest='create',
+                        action='store_true')
```

Comment:
nit: the option name doesn't make it obvious what's being created here, I suggest something slightly more verbose like `--create-user`